docs/source/getting_started.rst

```diff
@@ -1,66 +1,29 @@
-Getting Started
-===============
+Getting Started - Install
+=========================
 
 .. toctree::
    :maxdepth: 1
    :hidden:
 
    prereqs
    install
-   test_network
+   sdk_chaincode
 
-Before we begin, if you haven't already done so, you may wish to check that
-you have all the :doc:`prereqs` installed on the platform(s)
-on which you'll be developing blockchain applications and/or operating
-Hyperledger Fabric.
 
-Once you have the prerequisites installed, you are ready to download and
-install HyperLedger Fabric. While we work on developing real installers for the
-Fabric binaries, we provide a script that will :doc:`install` to your system.
-The script also will download the Docker images to your local registry.
+The Fabric application stack has 5 layers:
 
-After you have downloaded the Fabric Samples and Docker images to your local
-machine, you can get started working with Fabric with the
-:doc:`test_network` tutorial.
+.. image:: ./getting_started_image2.png
+   :width: 300px
+   :align: right
+   :height: 100px
+   :alt: Fabric Application Stack
 
-Hyperledger Fabric smart contract (chaincode) APIs
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+* :doc:`Prerequisite software <prereqs>` - the base layer needed to run the software, for example, Docker.
+* :doc:`Fabric and Fabric Samples <install>` - the Fabric executables to run a Fabric Network along with sample code.
+* :doc:`Contract APIs <sdk_chaincode>` - to develop smart contracts executed on a Fabric Network.
+* :doc:`Application SDKs <sdk_chaincode>` - to develop your blockchain application.
+* The Application - your blockchain application will utilize the Application SDKs to call smart contracts running on a Fabric Network.
```

Comment:
Why is "samples" and "network" capitalized here? And again, these hyphens would be better as colons.

Reply:
done.